include/mbedtls/config_adjust_legacy_crypto.h

```diff
@@ -48,7 +48,8 @@
     defined(MBEDTLS_PSA_BUILTIN_ALG_ECB_NO_PADDING) || \
     defined(MBEDTLS_PSA_BUILTIN_ALG_CBC_NO_PADDING) || \
     defined(MBEDTLS_PSA_BUILTIN_ALG_CBC_PKCS7) || \
-    defined(MBEDTLS_PSA_BUILTIN_ALG_CCM_STAR_NO_TAG))
+    defined(MBEDTLS_PSA_BUILTIN_ALG_CCM_STAR_NO_TAG) || \
+    defined(MBEDTLS_PSA_BUILTIN_ALG_CMAC))
 #define MBEDTLS_CIPHER_C
 #endif
 
```

Comment:
Changelog added in https://github.com/Mbed-TLS/mbedtls/pull/9178/commits/897ab810a762fee547679e391c3952fef28b37dd
For sake of completeness I also created an associated issue so that I could also mention it in the changelog.